stmt.go

```diff
@@ -56,21 +57,26 @@ func (s *stmt) QueryContext(ctx context.Context, args []driver.NamedValue) (driv
 	}
 
 	var it *spanner.RowIterator
-	if s.conn.roTx != nil {
-		it = s.conn.roTx.Query(ctx, ss)
-	} else if s.conn.rwTx != nil {
-		it = s.conn.rwTx.Query(ctx, ss)
+	if s.conn.tx != nil {
+		it = s.conn.tx.Query(ctx, ss)
 	} else {
 		it = s.conn.client.Single().Query(ctx, ss)
 	}
 	return &rows{it: it}, nil
 }
 
+func (s *stmt) CheckNamedValue(value *driver.NamedValue) error {
+	return nil
```

Comment:
So we do nothing here?

Reply:
Yes (at least for now). This method can be used by the driver to remove special parameters from a statement, or to do conversion for special parameters. This might be used in the future for things like for example parameters that should set the commit timestamp.